src/main/java/mServer/crawler/sender/newsearch/ZDFEntryTask.java

```diff
@@ -7,59 +7,78 @@
 import com.sun.jersey.api.client.WebResource;
 
 import java.util.concurrent.RecursiveTask;
+
 import mSearch.Config;
 import mSearch.tool.Log;
 
 /**
  * Searches all information required for a film
  */
-public class ZDFEntryTask extends RecursiveTask<VideoDTO> {
+public class ZDFEntryTask extends RecursiveTask<VideoDTO>
+{
 
     private static final long serialVersionUID = 1L;
-    
+
     private final ZDFClient client;
     private final ZDFEntryDTO zdfEntryDTO;
     private final Gson gson;
-    
-    public ZDFEntryTask(ZDFEntryDTO aEntryDto) {
+
+    public ZDFEntryTask(ZDFEntryDTO aEntryDto)
+    {
         client = new ZDFClient();
-        zdfEntryDTO = aEntryDto;                
+        zdfEntryDTO = aEntryDto;
         gson = new GsonBuilder()
                 .registerTypeAdapter(VideoDTO.class, new ZDFVideoDTODeserializer())
                 .registerTypeAdapter(DownloadDTO.class, new ZDFDownloadDTODeserializer())
                 .create();
     }
-    
+
     @Override
-    protected VideoDTO compute() {
+    protected VideoDTO compute()
+    {
 
         VideoDTO dto = null;
 
-        if(!Config.getStop()) {
-            try {
+        if (!Config.getStop())
+        {
+            try
+            {
                 // read film details
                 String infoUrl = zdfEntryDTO.getEntryGeneralInformationUrl();
                 WebResource webResourceInfo = client.createResource(infoUrl);
                 JsonObject baseObjectInfo = client.execute(webResourceInfo, ZDFClient.ZDFClientMode.VIDEO);
-                if(baseObjectInfo != null) {
+                if (baseObjectInfo != null)
+                {
                     dto = gson.fromJson(baseObjectInfo, VideoDTO.class);
-                    if(dto != null) {
+                    try
+                    {
+                        if (baseObjectInfo.getAsString().contains("Varg Venum"))
+                        {
+                            System.out.println("found varg venum");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)